.clang-format

```diff
@@ -1,2 +0,0 @@
-BasedOnStyle: WebKit
-Standard: "c++20"
```

Comment:
Can you add these back? They are used for the C++ code in the examples, otherwise editors will try and reformat in a different style

Reply:
added back, looks like only the .clangd file was causing this issue with C code:
<img width="500" alt="image" src="https://github.com/user-attachments/assets/45484b9e-d19b-43c8-b370-ffd2133bf35e" />